app/json/slate_configs.json

```diff
@@ -260,43 +260,6 @@
       }
     ]
   },
-  {
-    "id": "631d8077-1462-4397-ad0a-aa340c27570a",
-    "displayName": "Recommended reads, just for you",
-    "description": "Stories to fuel your mind, curated for your interests",
-    "internalDescription": "Curated Personalized",
-    "experiments": [
-      {
-        "description": "recit-personalized curated",
-        "candidateSets": [
-          "recit-personalized/curated"
-        ],
-        "rankers": [
-          "top30-items",
-          "thompson-sampling-28day",
-          "pubspread"
-        ]
-      }
-    ]
-  },
-  {
-    "id": "ad73c061-5ac5-458f-8691-ab071f7eadd7",
```

Comment:
[Github search shows no client usage](https://github.com/search?q=org%3APocket%20ab071f7eadd7&type=code)

Reply:
found 108 users who have accessed this slate in the last 90 days.  may also have older android clients:
```
select 
    e.hashed_user_id,
    e.slate_lineup_id,
    e.os,
    count(*) as eng
from engagements as e 
where e.slate_id in ('631d8077-1462-4397-ad0a-aa340c27570a', 'ad73c061-5ac5-458f-8691-ab071f7eadd7')
and e.happened_at > current_date - 90
and slate_lineup_id is not null
group by 1, 2, 3
```

Reply:
Thanks for remembering to check Snowflake! When I change the condition to `slate_lineup_id is null` I do not get any rows back, which means that clients only request these slates as part of a lineup. I double-checked that they've both been removed from slate_lineup_configs.json.